CHANGELOG.md

```diff
@@ -3,6 +3,124 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.2
+
+_Nov 23, 2023_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 📈 Export missing Charts props
```

Comment:
Do you see anything else worth highlighting?

Reply:
Nothing on pickers / tree view